evergreen/convert-client-cert-to-pkcs12.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with an error if any of the commands fail
+
+# Environment variables used as input:
+#   CLIENT_PEM                      Path to mongo -orchestration's client.pem: must be set.
+#   MONGO_X509_CLIENT_P12           filename for client certificate in p12 format
```

Comment:
I guess fileName should be started from the capital letter :) Similar below with password

Reply:
Done